app/views/content/fund-your-training.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Fund your training"
+heading: "Fund your training"
+description: |-
+    Find out about the financial support you can get to help pay for your trianing, including loans, scholarships and bursaries.
+title_paragraph: |-
+    You can get support while you are training, from a range of financial support to help pay your fees to practical assistance if you are disabled.
```

Comment:
If this section is now just 'funding your training', not 'funding and support', I'm not sure this description works?

Reply:
Let's discuss whether we need to go back to 'Funding and support' - I'm still in two minds. Perhaps something you could discuss with the TTAs on Wednesday?

'Fund your training' perhaps sounds a little bit like funding is available for everyone?